sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/cross/CrossProject.scala

```diff
@@ -290,6 +290,16 @@ object CrossProject extends CrossProjectExtra {
 
   def apply(jvmId: String, jsId: String, base: File,
       crossType: CrossType): CrossProject = {
+    val crossProject = withOutScalaJSPlugin(jvmId, jsId, base, crossType)
+    crossProject.js.enablePlugins(ScalaJSPlugin)
+    crossProject
+  }
+
+  /** This is an unsafe variant of CrossProject.apply that assumes that the
+   *  ScalaJSPlugin will be added manually later on.
+   */
+  def withOutScalaJSPlugin(jvmId: String, jsId: String, base: File,
```

Comment:
without is written as a single word in english. So I would not camel case there.